x-pack/test/case_api_integration/security_only/tests/common/comments/delete_comment.ts

```diff
@@ -170,38 +171,6 @@ export default ({ getService }: FtrProviderContext): void => {
       });
     }
 
-    it('should not delete a comment with no kibana privileges', async () => {
```

Comment:
Why this is deleted?

Reply:
oh yeah, I should have commented on this haha. Yeah this was actually a bug I think haha. I moved the `noKibanaPrivileges` user up into the test case above this because it's exactly the same as the other users. I think the reason we had to split it out before was because the way we had the security plugin still managing the saved object permissions. This was causing a 404 on the delete all comments instead of a 403 which is what all the other users were causing. The only difference between this test and the one above is that the delete all comments was causing a 404 instead of the 403.